pennylane_ionq/device.py

```diff
@@ -109,16 +123,25 @@ def reset(self):
         self._prob_array = None
         self.histogram = None
         self.circuit = {
+            "format": "ionq.circuit.v0",
             "qubits": self.num_wires,
             "circuit": [],
             "gateset": self.gateset,
         }
         self.job = {
-            "lang": "json",
-            "body": self.circuit,
+            "input": self.circuit,
             "target": self.target,
             "shots": self.shots,
         }
+        if self.error_mitigation is not None:
+            self.job["error_mitigation"] = self.error_mitigation
+        if self.job["target"] == "qpu":
+            self.job["target"] = "qpu.harmony"
+            warnings.warn(
+                "The ionq_qpu backend is deprecated. Defaulting to ionq_qpu.harmony.",
+                DeprecationWarning,
+                stacklevel=2,

```

Comment:
we tend to use `UserWarning` because `DeprecationWarning` doesn't appear in some environments. If we just left the target as `qpu`, would this swap happen on the IonQ side? Also, where do you intend on the user seeing this coming from/what motivates the `stacklevel=2`?
```suggestion
                UserWarning,
                stacklevel=2,
```